src/commands/status.js

```diff
@@ -0,0 +1,86 @@
+const os = require('os');
+const publicip = require('public-ip');
+const prettyms = require('pretty-ms');
+const prettybyte = require('pretty-bytes');
+const {table} = require('table');
+
+const BaseCommand = require('../oclif/command/BaseCommand');
+
+const MuteOneLineError = require('../oclif/errors/MuteOneLineError');
+
+const PRESETS = require('../presets');
+
+class StatusCommand extends BaseCommand {
+  /**
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {DockerCompose} dockerCompose
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    {
+      preset, 'system': targetSystem,
+    },
+    flags,
+    dockerCompose,
+  ) {
+    const data = [];
+    let output;
+
+    const tableConfig = {
+      //singleLine: true,
+      drawHorizontalLine: (index, size) => {
+        return index === 0 || index === 1 || index === size;
+      }
+    };
+
+    if (targetSystem === 'host') {
```

Comment:
Maybe it's better to do nested commands instead of params?

Reply:
You mean like `mn status:(host|docker|masternode)` right? I think that makes so much more sense, thanks for the suggestion! Does this look like it is implemented correctly?

Reply:
It looks good! Will have a summary status command `mn status`?

Reply:
Ah, now I understand what the oclif documentation means with "topics"! I've added a command showing some basic status information for `mn status`. The nested command classes should still extend `BaseCommand` right?

Let's continue discussion in #17 with more people about what they would like each command in the topic to display? I would add this in smaller future PRs.

Reply:
> The nested command classes should still extend BaseCommand right?
yes, I think so.